docs/containers-storage.conf.5.md

```diff
@@ -145,7 +145,11 @@ The `storage.options.overlay` table supports the following options:
   ignore_chown_errors can be set to allow a non privileged user running with a  single UID within a user namespace to run containers. The user can pull and use any image even those with multiple uids.  Note multiple UIDs will be squashed down to the default uid in the container.  These images will have no separation between the users in the container. (default: false)
 
 **mount_program**=""
-  Specifies the path to a custom program to use instead of using kernel defaults for mounting the file system.
+  Specifies the path to a custom program to use instead of using kernel defaults
+for mounting the file system. In rootless mode, no CAP_SYS_ADMIN capability,
```

Comment:
s/no CAP_SYS_ADMIN/without the CAP_SYS_ADMIN/? It's a bit hard to brain parse for me.

Reply:
Fixed